doc/decoding/decoding_intro.rst

```diff
@@ -463,13 +463,13 @@ We can try Fisher's `Linear Discriminant Analysis (LDA)
 
 Import the module::
 
-    >>> from sklearn.lda import LDA
+    >>> from sklearn.lda import LDA  # doctest: +SKIP
```

Comment:
Done. It looks like this
![lda](https://user-images.githubusercontent.com/11410385/29529889-c1e6c738-86a1-11e7-9ab9-2fab5e7b235f.png)